scripts/ci/copy-env-vars-to-dot-env-file

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -eux
+
+echo "NUXT_R_API_BASE=$NUXT_R_API_BASE" > .env
```

Comment:
This file is a bit of infrastructure to make it convenient to add lots more env vars at once to GitHub Actions much easily (assuming we'll get more of them in the future)